### PR TITLE
Documenting the Descheduler Operator for 4.0

### PR DESCRIPTION
**Pull request by mburke5678**
Includes: https://bugzilla.redhat.com/show_bug.cgi?id=1684445

### CODE REVIEW

File: modules/nodes-scheduler-descheduler-about.adoc [hunk 1]
```diff
@@ -55,48 +48,13 @@ an eviction policy.
...
 ====
 
-The following sections describe the process to configure and run the descheduler:
+The following is a high-level process to configure and run the Descheduler Operator:
 
 * Create a role with the needed permissions. 
 
 * Define the descheduling behavior in a policy file. 
 
-* Create a configuration map to reference the policy file.
```
Comment (mburke5678) [inline modules/nodes-scheduler-descheduler-about.adoc]: Moved this section to _about_.

**Comment by mburke5678** [comment]
@joelsmith PTAL

**Comment by mburke5678** [comment]
@ravisantoshgudimetla PTAL

cc. @jcantrill

File: _topic_map.yml [hunk 2]
```diff
@@ -348,7 +348,7 @@ Topics:
     File: nodes-scheduler-taints-tolerations
   - Name: Constraining pod placement using node selectors
     File: nodes-scheduler-node-selectors
-  - Name: Keeping your cluster balanced using the descheduler
+  - Name: Keeping your cluster balanced using the Descheduler Operator
```
Comment (ravisantoshgudimetla) [inline _topic_map.yml]: @mburke5678 - Descheduler is still reponsible for balancing the cluster, the management of descheduler now happen through descheduler operator.

File: modules/nodes-scheduler-descheduler-about.adoc [hunk 3]
```diff
@@ -3,32 +3,25 @@
 // * nodes/nodes-scheduler-descheduler.adoc
 
 [id='nodes-scheduler-descheduling-about_{context}']
-= Understanding how to evict pods in {product-title}
+= Understanding how to evict pods with the Descheduler Operator
```
Comment (ravisantoshgudimetla) [inline modules/nodes-scheduler-descheduler-about.adoc]: Please change everything back to `Descheduler`

File: modules/nodes-scheduler-descheduler-about.adoc [hunk 4]
```diff
@@ -3,32 +3,25 @@
...
 
 Because {product-title} clusters are highly dynamic and their state changes over time 
-your cluster can benefit from descheduling pods from one node and rescheduling onto a more appropriate node for various reasons:
+your cluster can benefit from installing and using the Descheduler Operator to deschedule pods from one node and rescheduling onto a more appropriate node for various reasons:
```
Comment (ravisantoshgudimetla) [inline modules/nodes-scheduler-descheduler-about.adoc]: s/to deschedule pods/to manage descheduler which can evict

File: modules/nodes-scheduler-descheduler-create-role.adoc [hunk 5]
```diff
@@ -38,27 +38,27 @@ rules:
 . Create the service account that runs the job:
 +
 ----
-# oc create sa <file-name>.yaml -n kube-system
+# oc create sa <file-name>.yaml -n openshift-operators
```
Comment (ravisantoshgudimetla) [inline modules/nodes-scheduler-descheduler-create-role.adoc]: IIRC, it should be in descheduler-operator project

File: modules/nodes-scheduler-descheduler-policy.adoc [hunk 6]
```diff
@@ -0,0 +1,93 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-scheduler-descheduler.adoc
+
+[id='nodes-scheduler-descheduling-policy_{context}']
+= Understanding descheduler policies
+
```
Comment (ravisantoshgudimetla) [inline modules/nodes-scheduler-descheduler-policy.adoc]: This section lgtm

**Review by ravisantoshgudimetla — COMMENTED** [review]
@mburke5678 - Many thanks for the PR, please look at my comments and we can merge once those are addressed.

**Comment by mburke5678** [comment]
@ravisantoshgudimetla Thank you for your comments!!

File: modules/nodes-scheduler-descheduler-create-role.adoc (hunk 5 above)
Comment (mburke5678) [reply modules/nodes-scheduler-descheduler-create-role.adoc]: @ravisantoshgudimetla I am seeing that the Descheduler Operator and CRD are defaulting to the `openshift-operators` project. I just did an install.

**Comment by mburke5678** [comment]
@jhou1 Can you take a look at this PR or assign to another QE person?

**Comment by jhou1** [comment]
@sunilcio PTAL

**Pull request merged**